### ISSUE TITLE
Cpp runtime on Windows can't handle spaces in install path

### ISSUE THREAD

**Issue by OpenModelica-TracImporter** [OpenModelica/OpenModelica#6004]

**Comment by NikolaiVChr** [comment]
This is still an issue in v1.23.0
![image](https://github.com/OpenModelica/OpenModelica/assets/3764194/2f23043d-9dc9-4408-b4bd-23f8e224afee)

**Comment by casella** [comment]
What a nightmare...

@adrpo, any ideas?

**Comment by adrpo** [comment]
This has nothing to do with the CPP runtime. It is the regular runtime. 

Our fixes for spaces in paths is translating the path with spaces to short paths but if they are not activated they do not work.
One would need to enable it via an administrator command prompt: 

`fsutil 8dot3name set D: 0`


You can check if is activated via: 

`fsutil 8dot3name query D:`.

**Comment by adrpo** [comment]
True, not everybody has admin rights. We should check if the path as spaces in it and quote it in OMEdit when we call the Compile.bat script.

**Comment by bilderbuchi** [comment]
You don't have to even check, and could just quote unconditionally, right -- no harm done to non-spaced paths, and you'll have simpler logic/less complexity that way?

**Comment by adrpo** [comment]
Sure. Will do. I need to check how that goes into the Makefile we generate to be sure it works.
